### PR TITLE
Add `CAMERA_VISIBLE_LAYERS` to spatial shader docs

### PR DESCRIPTION
**Pull request by xXAdrXx**
Added Documentation for CAMERA_VISIBLE_LAYERS

_bugsquad edit: Fixes https://github.com/godotengine/godot-docs/issues/8482_

### CODE REVIEW

File: tutorials/shaders/shader_reference/spatial_shader.rst [hunk 1]
```diff
@@ -172,6 +172,8 @@ shader, this value can be used as desired.
 +----------------------------------------+--------------------------------------------------------+
 | in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction.                          |
 +----------------------------------------+--------------------------------------------------------+
+| in int **CAMERA_VISIBLE_LAYERS**        | Visible layers indicator for the camera.              |
```
Comment (mhilbrunner) [inline tutorials/shaders/shader_reference/spatial_shader.rst:175]:
```suggestion
| in int **CAMERA_VISIBLE_LAYERS**       | Visible layers indicator for the camera.               |
```

Comment (clayjohn) [reply tutorials/shaders/shader_reference/spatial_shader.rst:175]: I would reword the description as well to say "Cull layers of the camera rendering the current pass." instead.

**Review by AThousandShips — APPROVED** [review]

**Comment by AThousandShips** [comment]
Thanks! And congratulations on your first merged commit! 🎉

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2 in https://github.com/godotengine/godot-docs/pull/9648.
